scripts/superdesk-authoring/versioning/versions/views/versions.html

```diff
@@ -1,16 +1,6 @@
 <div class="article-versions" ng-controller="VersioningWidgetCtrl">
     <ul class="versions-list card-list">
-        <li class="item draft" ng-class="{selected: selected === item._autosave}" ng-if="item._autosave" ng-click="openAutosave()">
```

Comment:
so there won't be any indication that you have made some changes? or am I missing something